TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -157,15 +157,15 @@ def makeTreeFromMiniAOD(self,process):
                 genCollection = cms.InputTag("prunedGenParticles"),
                 debug = cms.bool(False),
                 # Particles we want to save from the decay chain of the tops
-                childIds = cms.vint32(1,2,3,4,5,11,12,13,14,15,16,24),
+                childIds = cms.vint32(1,2,3,4,5,6,11,12,13,14,15,16,24),
```

Comment:
In the past we have not added the top quark ID as one of the child IDs to save because the top is not a final state particle. Is there a reason why you are adding it here? Have you checked what affect this will have on TTJets samples and signal samples? My guess is that it will have little to no effect on the signal samples, but will start to save a bunch of extra particles for TTJets or T1tttt/T2tt samples.

Reply:
That was mainly due to concerns about the top type coupling of the flavored model. I preemptively added this without thinking too much about what this would do to other samples, but after testing with our private samples, I don't think we need this change. I will be removing this in the next update